vllm/attention/backends/abstract.py

```diff
@@ -105,6 +105,10 @@ class AttentionMetadata:
     # in block 0, and 1st slot in block 1, respectively.
     slot_mapping: torch.Tensor
 
+    # Number of original input tokens (without any decoding).
+    # Some model (phi3-) need this info to decide model settings
+    num_orig_input_tokens_tensor: torch.Tensor
```

Comment:
Is this actually `Optional[torch.Tensor]`?

Reply:
@robertgshaw2-neuralmagic thanks for the suggestion! Fixed it